config/initializers/acl.rb

```diff
@@ -0,0 +1,25 @@
+module ACL
+  extend self
+  ADMIN_USERS = {}
+
+  # Given ENV['ADMIN_USERS'] #=> "qrush=nick@example.com:bf4=bf@example.com"
```

Comment:
I'd be OK with hard coding this for now in a `config` file if we need to. `config_for` is super helpful for this stuff and it'll let us test stuff too:

http://api.rubyonrails.org/classes/Rails/Application.html#method-i-config_for

Let's do myself, @bf4, @dwradcliffe to start.